### ISSUE TITLE
default value parsing error

### ISSUE THREAD

**Issue by myyrakle** [oguimbal/pgsql-ast-parser#53]
I entered and tested a valid postgresql ddl query, but there is a case where the value of the default clause cannot be read normally.

The test case is:
```
CREATE TABLE "tb_user" (
	"user_no"	serial8		NOT NULL,
	"reg_date"	timestamptz	DEFAULT (CURRENT_TIMESTAMP AT TIME ZONE 'Asia/Seoul')	NOT NULL,
	"complete_yn"	boolean	DEFAULT false	NOT NULL
);
```

```
const foo = parseWithComments(sql);
console.log(foo);
```

**Comment by oguimbal** [comment]
Hi,

Sorry for the delay, I dont have enough time to work on this these days.

I'll try to have a look when I have time.

Thanks for the report, and for the kind words

**Comment by oguimbal** [comment]
Hi ! Fixed in `pgsql-ast-parser@9.1.0`. Sorry for the delay :)

**Comment by myyrakle** [comment]
Thank you for your effort.
